src/components/SnippetCard/index.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import AceEditor from 'react-ace';
+
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+
+import { Button, Card, message, Modal, Typography } from 'antd';
+
+import 'ace-builds/src-noconflict/mode-c_cpp';
```

Comment:
lets put this in a separate component. along with the actual <AceEditor /> component below